packages/frontend-shared/README.md

```diff
@@ -1,43 +1,160 @@
 ## from repo root
 yarn workspace @packages/frontend-shared cypress:open
 ```
 
 ## Developing
-
 For the best development experience, you will want to use VS Code with the [Volar](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar) extension. This will give you type completion inside `vue` files.
 
 ## Testing
-
 ```bash
-yarn workspace @packages/frontend-shared test-unit
+## from repo root
+yarn workspace @packages/frontend-shared cypress:run:ct
 ```
 
 ## Utility class usage
-
-Windi CSS can create an awesome interactive summary showing our usage of utility classes and design tokens. Running this command will generate this report and serve it on localhost.
+WindiCSS can create an awesome interactive summary showing our usage of utility classes and design tokens. Running this command will generate this report and serve it on localhost.
 
 ```bash
 ## from this directory
 yarn windi
 ```
 
-## Use of graphql by shared components is limited to `src/gql-components`
-
-In the long run, files in the `src/components` directory are intended as the foundation of a design system. As such they may be used in many contexts other than the Cypress App and Launchpad. There are some components that are only intended to be shared between App and Launchpad and make use of GraphQL queries and mutations. These will only work correctly if placed within `src/gql-components` directory.
+This will be useful from time to time so that we can audit our usage of these classes and extract repeated patterns into Windi shortcuts or otherwise consolidate them, when it makes sense to do so.
 
 ## Link Components
-
 There are two shared components involved with links - `BaseLink`, and `ExternalLink`. `BaseLink` is responsible for default colors and hover/focus styles. `ExternalLink` wraps `BaseLink` is responsible for managing the GraphQL mutation that triggers links to open the in the user's default browser.
 
 ## Generate the theme for shiki
-
 See [the readme in the src/public/shiki/themes directory](./src/public/shiki/themes/ReadMe.md)
+
+## Front-end Conventions, Underlying Ideas, and Gotchas
+These apply to this package, `app`, and `launchpad`, as well as any future work in this Vue-Tailwind-GQL stack. The goal is for this to provide useful context for new developers adding features to the codebase, or making changes to existing features. There are pros and cons to all of these decisions, but rather than get into those in detail, this is just a document of what practices we are following.
+
+### Development Workflow
+We recommend component-based test driven development. More details in the [Testing Practices](../../guides/testing-strategy-and-styleguide.md) guide. To make changes to an existing component:
+
+1. Open Cypress and got to the spec that covers the component (often it's 1:1 but sometimes components are tested via their parents)
+1. Update the test to reflect the desired change (or part of it)
+1. Implement the Change in the component
```

Comment:
I'm not entirely sure if "the Change" here should be written with an uppercase letter.

Reply:
Good eyes on all these :)